src/licensedcode/data/licenses/ecfonts-1.0.yml

```diff
@@ -0,0 +1,8 @@
+key: ecfonts-1.0
+shortname: latex-ec-fonts
+name: Copyright notice to the ec fonts
+category: Proprietary Free
```

Comment:
The category is really simply "Permissive"   the extra text being simply detailed obligations to anyone using the software under this license.